samples/word/04-range/scroll-to-range.yaml

```diff
@@ -4,47 +4,55 @@ description: Shows how to scroll to a range with and without selection.
 author: OfficeDev
 host: WORD
 api_set:
-    WordApi: 1.1
+    WordApi: 1.2
 script:
-    content: |
-        $("#setup").click(setup);
-        $("#scroll").click(scroll);
-        $("#scroll-end").click(scrollEnd);
+    content: |-
+        $("#setup").click(() => tryCatch(setup));
+        $("#scroll").click(() => tryCatch(scroll));
+        $("#scroll-end").click(() => tryCatch(scrollEnd));
 
-        function scroll() {
-            Word.run(function (context) {
+
+        async function scroll() {
+            await Word.run(async (context) => {
                 //if select is called with no parameters it selects the object.
-                var paragraph = context.document.body.paragraphs.getLast().select();
-                return context.sync()
-                    .then(function () {
-                        console.log("success");
-                    });
-            })
-                .catch(OfficeHelpers.Utilities.log);
+                context.document.body.paragraphs.getLast().select();
+
+                await context.sync();
+                console.log("success");
```

Comment:
One of multiple places in this file, where the code is logging success and I don't think it's really necessary.

Reply:
local fixed.